java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -302,37 +227,13 @@ private synchronized ColumnVector incRefCountInternal(boolean isFirstTime) {
     return this;
   }
 
-  /**
-   * Returns a new ColumnVector with NaNs converted to nulls, preserving the existing null values.
-   */
-  public ColumnVector nansToNulls() {
-    assert type == DType.FLOAT32 || type == DType.FLOAT64;
-    return new ColumnVector(nansToNulls(this.getNativeView()));
-  }
-
-  /**
-   * Returns the number of rows in this vector.
-   */
-  @Override
-  public long getRowCount() {
-    return rows;
-  }
-
   /**
    * Returns the amount of device memory used.
    */
   public long getDeviceMemorySize() {
```

Comment:
This should move to `ColumnView` since it operates on a view after #6786

Reply:
done.